crates/bevy_pbr/src/meshlet/from_mesh.rs

```diff
@@ -306,7 +290,8 @@ fn simplify_meshlet_groups(
 
     // Allow more deformation for high LOD levels (1% at LOD 1, 10% at LOD 20+)
     let t = (lod_level - 1) as f32 / 19.0;
-    let target_error = 0.1 * t + 0.01 * (1.0 - t);
+    let target_error_rel = 0.1 * t + 0.01 * (1.0 - t);
```

Comment:
Small nit: please write out full variable names, and don't abbreviate it to rel

Reply:
Renamed to target_error_relative